packages/rum-core/src/domain/view/viewMetrics/trackInteractionToNextPaint.ts

```diff
@@ -74,6 +73,10 @@ export function trackInteractionToNextPaint(
           configuration.actionNameAttribute
         )
       } else {
+        addTelemetryDebug('INP target is null or not an element node', {
+          target: JSON.stringify(newInteraction.target), // if target is not an element node, log it to help debugging
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** JSON.stringify(newInteraction.target) will always return '{}'. Maybe you could do it like so:
```ts
{
hasTarget: !!newInteraction.target
targetIsConnected: newInteraction.target?.isConnected,
...
}
```
‏